libs/cdk/coercing/spec/coerceDistinctObservableWith.spec.ts

```diff
@@ -320,18 +344,23 @@ describe('coerceDistinctWith', () => {
 
     it('should only emit 1 value since the coerced value for the first emission never completes', () => {
       testScheduler.run(({ cold, expectObservable, expectSubscriptions }) => {
-        const source =     '-a-b----c-d--------|'
-        const expected =   '-a------------------';
+        const source = '-a-b----c-d--------|';
+        const expected = '-a------------------';
         const valueSubs = {
-          b:               '-----',
-          c:               '-----',
-          d:               '-----',
+          b: '-----',
+          c: '-----',
+          d: '-----',
         };
 
         const values = {
-          a: 'hello dear contributor',
+          a: cold('(a)', { a: 'hello dear contributor' }),
```

Comment:
@BioPhoton this test was failing for a good reason and I don't understand why it passed before this upgrade.

`coerceDistinctWith` internally uses `coerceObservableWith` which will turn any non-observable value into `of`.

`of` completes immediately, so `exhaustAll()` emits values.

Do we want to change that behavior? For now, i've adjusted the spec

Reply:
Assuming we keep the spec adjusted and move forward.